logging-sinks/src/worker.ts

```diff
@@ -1,9 +1,10 @@
 // @@@SNIPSTART typescript-logger-sink-worker
-import { Worker, InjectedSinks } from '@temporalio/worker';
-import { LoggerSinks } from './workflows';
+import { defaultSinks, InjectedSinks, LoggerSinks, Worker } from '@temporalio/worker';
+import { CustomLoggerSinks } from './workflows';
 
 async function main() {
-  const sinks: InjectedSinks<LoggerSinks> = {
+  const sinks: InjectedSinks<LoggerSinks & CustomLoggerSinks> = {
```

Comment:
@bergundy Alternatively, SDK could export `LoggerSinks` from `@temporalio/workflow`, and then user can do 

```ts
export type AllSinks = LoggerSinks & MySinks
```

in workflow code, and use it both in workflow and here in worker code.

Reply:
Good idea